tests/unit/test_transaction.py

```diff
@@ -587,6 +666,18 @@ def _execute_update_helper(
 
         self.assertEqual(transaction._execute_sql_count, count + 1)
 
+        if not HAS_OPENTELEMETRY_INSTALLED:
+            pass
+
+        span_list = self.get_finished_spans()
+        got_span_names = [span.name for span in span_list]
+        want_span_names = ["CloudSpanner.Transaction.execute_update"]
+        assert got_span_names == want_span_names
+
+        got_span_events_statuses = self.finished_spans_events_statuses()
+        want_span_events_statuses = []
+        assert got_span_events_statuses == want_span_events_statuses
```

Comment:
can we use self.assertSpanAttributes, since there are no span events to check for here?